projects/packages/connection/src/class-rest-connector.php

```diff
@@ -947,6 +986,40 @@ public static function disconnect_site() {
 		);
 	}
 
+	/**
+	 * Unlinks current user from the WordPress.com Servers.
+	 *
+	 * @since 6.3.3
+	 *
+	 * @param WP_REST_Request $request The request sent to the WP REST API.
+	 *
+	 * @return bool|WP_Error True if user successfully unlinked.
+	 */
+	public static function unlink_user( $request ) {
+
+		if ( ! isset( $request['linked'] ) || false !== $request['linked'] ) {
```

Comment:
I'm not running into issues while testing, so I assume this works correctly. I'm just having a hard time understanding why the correct thing to do is return an error if the user *is* linked in this case 🤔 

Wouldn't `false !== $request['linked']` be true if the user is linked? Or am I misunderstanding the `linked` parameter?

Reply:
It's kind of an odd parameter - if it's `false` it means we want the user to be unlinked from what I understand. At least that's the effect. It doesn't have to do with checking that the current user is connected already or not.